native/explorer/src/lazyframe/io.rs

```diff
@@ -54,6 +55,18 @@ pub fn lf_from_parquet_cloud(
     Ok(ExLazyFrame::new(lf))
 }
 
+#[cfg(not(feature = "aws"))]
```

Comment:
Why do we want to have this optional? Some targets do not support it as well?

Reply:
Yeah, the problem is that `object_store` has a dependency - called [ring](https://github.com/briansmith/ring) - that has some low-level code in C and Assembly which does not compile for RISCV computers. In my research I found that there were some attempts to fix the issue, but they didn't succeed. I was considering keep a fork and try some patches, but I think that would require some great amount of work.